### PR TITLE
Fix/avatar should light up brownie

### LINKED ISSUE #10581: Avatar should light up the brownie score on hover
Currently, if you hover over the user's avatar picture, it acts as if it was a different link than the brownie score. They both technically make part of the same button, therefor the behaviour should be the same.

screen shot 2016-09-09 at 10 31 24

Easy win. Who's up for it?

### PR DESCRIPTION
**Pull request by ooade**
#### Pre-Submission Checklist

- [ ] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [ ] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [ ] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #10581
#### Description

Merged `Avatar-Nav-Item` and `Points-Nav-Item` since they both navigate to /settings and besides, hovering on Avatar should light up the brownie score
changes

### CODE REVIEW

File: common/app/components/Nav/Avatar-Points-Nav-Item.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+
+// this is separated out to prevent react bootstrap's
+// NavBar from injecting unknown props to the li component
+export default function AvatarPointsNavItem({ picture, points }) {
+  return (
+    <li className='avatar-points' key='user'>
```
Comment (BerkeleyTrue) [inline common/app/components/Nav/Avatar-Points-Nav-Item.jsx]: multi-attribute jsx should be spread out one attribute per line

**Comment by BerkeleyTrue** [comment]
You have extra commits. Please clean up your commits.

File: common/app/components/Nav/Nav.jsx [hunk 2]
```diff
@@ -163,30 +162,12 @@ export default class extends React.Component {
...
-  }
-
-  renderSignIn(username, picture, shouldShowSignIn) {
+  renderSignIn(username, points, picture, shouldShowSignIn) {
     if (!shouldShowSignIn) {
       return null;
     }
     if (username) {
-      return <AvatarNavItem picture={ picture } />;
+      return <AvatarPointsNavItem picture={ picture } points={ points } />;
```
Comment (BerkeleyTrue) [inline common/app/components/Nav/Nav.jsx]: multi-attribute jsx should be spread out one attribute per line

**Comment by marhyorh** [comment]
I ammend them. Thanks!

**Comment by dhcodes** [comment]
@marhyorh your changes appear to incorporate commits from other users; you might need to rebase your branch and then amend the pr.

**Comment by marhyorh** [comment]
I moved this to https://github.com/FreeCodeCamp/FreeCodeCamp/pull/10895 inorder to discard those commit changes that were not mine :smile: @dhcodes @hallaathrad

**Pull request closed without merging**